modules/react/text-input/lib/InputGroup.tsx

```diff
@@ -164,5 +239,10 @@ export const InputGroup = createComponent('div')({
      * within the input.
      */
     InnerEnd: InputGroupInnerEnd,
+    /**
+     * A component that can be added to an input group that will clear the input. It will only render
+     * when the input has a value and will fade when a value is entered.
+     */
+    ClearInputButton: ClearInputButton,
```

Comment:
I'm fine with this or `ClearButton` since it already has `InputGroup` in front when you use it. `InputGroup.ClearButton` . This is super nit, non blocking